### ISSUE TITLE
Very poor performance on pre-trained models.. maybe un matching Labels to index file 

### ISSUE THREAD

**Issue by Purav-Zumkhawala** [kenshohara/3D-ResNets-PyTorch#243]
Hi, 
I am trying to test the r3d50_K_200ep.pth model on youtube videos and the pre-trained model is predicting very poorly.. I am not getting any correct predictions which make me wonder that the class list mapping to index that I have, is not the same one that was used while training. I am using the kinetics-700 class list in sorted order. The command I am using is the following for generating the inferences

`"python main.py --root_path "+root_path+" --video_path mini_clips_extracted_jpg\\Shaving --annotation_path test.json "+"--result_path results --dataset kinetics --resume_path "+model_path+" --model_depth 50 --n_classes 700 --no_train --no_val --inference --output_topk 1 --inference_batch_size 1"`

I have used the same command for predicting on the UCF101 dataset after fine-tuning it on the kinetic-400 pre-trained model and it performs really well like given in the papers.

Does anyone have any idea what might be causing these issues. 
@kenshohara @guilhermesurek

**Comment by Purav-Zumkhawala** [comment]
The issue was as I suspected the class labels while training and the ones that I was using the sorted labels were not matching and thus I was getting low accuracy. 

Look at this issue for the labels that were used for training. The answer is given by @Jimmy880.
Issue link: #211
